Food Ordering System/style.css

```diff
@@ -0,0 +1,325 @@
+*{
+    font-family: 'Poppins', sans-serif;
+}
+
+body {
+    margin: 0 auto 100%;
+    padding: 0;
+    overflow-x: hidden;
+    width: 100%;
+    height: 100%;
+}
+/* Header */
```

Comment:
@andreeaiulia13 if you feel the need to add a comment here it must mean that the name of the class is not explainatory enough and you should change that + remove the comment :-)